test-suite-graal/test-suite-graal-jte/build.gradle

```diff
@@ -11,7 +11,6 @@ dependencies {
 jte {
     sourceDirectory = file('src/main/resources/views').toPath()
     generate()
-    generateNativeImageResources = false
```

Comment:
why are you doing this?

Reply:
Because this configuration does not exist in newer versions of jte (see "Breaking changes" section in the [release notes](https://github.com/casid/jte/releases/tag/3.0.0)). This was also turning off jte's native image generation feature, so there is no need to add the new way of enabling it.